daemon/update_linux.go

```diff
@@ -23,7 +23,7 @@ func toContainerdResources(resources container.Resources) libcontainerd.Resource
 	r.CpusetCpus = resources.CpusetCpus
 	r.CpusetMems = resources.CpusetMems
 	r.MemoryLimit = uint64(resources.Memory)
-	if resources.MemorySwap > 0 {
+	if resources.MemorySwap != 0 {
```

Comment:
I don't know if that's necessary, other invalid values are already covered in `verifyContainerSettings`, so there won't be other values here.